core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSource.scala

```diff
@@ -1011,12 +1011,12 @@ case class DeltaSource(
     var commitProcessedInBatch = false
 
     /**
-     * This overloaded method checks if all the AddCDCFiles for a commit can be accommodated by
+     * This overloaded method checks if all the FileActions for a commit can be accommodated by
```

Comment:
nit: Remove "This overloaded method"

Move the description to be `@return` and describe what it does (_returns_).